Formula/libmpdec.rb

```diff
@@ -0,0 +1,41 @@
+class Libmpdec < Formula
+  desc "Library for decimal floating point arithmetic"
+  homepage "https://www.bytereef.org/mpdecimal/"
+  url "https://www.bytereef.org/software/mpdecimal/releases/mpdecimal-2.5.0.tar.gz"
+  sha256 "15417edc8e12a57d1d9d75fa7e3f22b158a3b98f44db9d694cfd2acde8dfa0ca"
+  license "BSD-2-Clause"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <mpdecimal.h>
+      #include <string.h>
```

Comment:
I think you also need an `#include <assert.h>` here.

Reply:
Yes, copy/paste error. Thank you.